Bluepill-cli/Bluepill-cli/Simulator/BPSimulator.m

```diff
@@ -186,8 +152,8 @@ - (SimDevice *)findDeviceWithConfig:(BPConfiguration *)config andDeviceID:(NSUUI
     return device; //could be nil when not found
  }
 
-- (NSRunningApplication *)findSimGUIAppWithDeviceUDID:(NSString *)deviceUDID {
-    NSString * cmd = [NSString stringWithFormat:@"ps -A | grep 'Simulator\\.app.*-CurrentDeviceUDID %@'", deviceUDID];
+- (NSRunningApplication *)findSimGUIApp {
```

Comment:
why do you remove deviceUDID? will sim reuse still work?

Reply:
simulator.app does not corresponds a certain simulator device any more. Removing deviceUDID is needed to make sim reuse work